initfiles/componentfiles/configschema/xsd/roxie.xsd

```diff
@@ -0,0 +1,640 @@
+                                  hpcc:tooltip="Turns on/off Priority weight-based for output queues (0 round-robin no priority - old logic, 1 round-robin new logic, 2 and higher is factor of priority)"/>
+                    <xs:attribute name="udpQueueSize" type="xs:nonNegativeInteger" hpcc:displayName="UDP Queue Size"
+                                  hpcc:presetValue="100" hpcc:tooltip="UDP transport layer receive queue size"/>
+                    <xs:attribute name="udpRequestToSendTimeout" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="UDP Request To Send Timeout (units?)" hpcc:presetValue="0"
+                                  hpcc:tooltip="Controls the timeout value agent udp will wait for permission to send from a Roxie server, in milliseconds. Specify 0 to calcuate automatically"/>
+                    <xs:attribute name="udpResendEnabled" type="xs:boolean" hpcc:displayName="Enable UDP Resend"
+                                  hpcc:presetValue="false" hpcc:tooltip="UDP transport layer packet resend ability"/>
+                    <xs:attribute name="udpRetryBusySenders" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="UDP Retry Busy Senders" hpcc:presetValue="0"
+                                  hpcc:tooltip="Controls the number of times Roxie server will repeat search for an idle sender when requesting new data"/>
+                    <xs:attribute name="udpSendQueueSize" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="UDP Send Queue Size" hpcc:presetValue="50"
+                                  hpcc:tooltip="UDP transport layer send queue size"/>
+                    <xs:attribute name="udpSnifferEnabled" type="xs:boolean" hpcc:displayName="Enable UDP Sniffer"
+                                  hpcc:presetValue="true"
+                                  hpcc:tooltip="Enable the UDP multicast sniffer for tracking which senders are busy"/>
+                    <xs:attribute name="udpSnifferReadThreadPriority" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="UDP Sniffer Read Thread Priority" hpcc:presetValue="3"
+                                  hpcc:tooltip="If non-zero, run the sniffer read thread at elevated priority level"/>
+                    <xs:attribute name="udpSnifferSendThreadPriority" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="UDP Sniffer Send Thread Priority" hpcc:presetValue="3"
+                                  hpcc:tooltip="If non-zero, run the sniffer send thread at elevated priority level"/>
+                </xs:attributeGroup>
+
+                <xs:attributeGroup name="cache" hpcc:groupByName="Cache">
+                    <xs:attribute name="blobCacheMem" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="Blob Cache Memory Size (Mb)" hpcc:presetValue="0"
```

Comment:
Why 0? No cache?

Reply:
This PR is only about conversion to the new format. A separate JIRA will go to Richard to review for correctness. At this point the values are strictly copied from the existing XSD.